.github/ServiceX/Dockerfile

```diff
@@ -0,0 +1,39 @@
+# Make the base image configurable:
+ARG BASEIMAGE=gitlab-registry.cern.ch/treemaker/cmssw-docker/cmssw:CMSSW_10_2_21-2020-04-07-fc019a29
+
+# Set up the CMSSW base:
+FROM ${BASEIMAGE}
+
+ARG BUILD_DATE
+ARG VCS_REF
+ARG VCS_URL
+ARG VERSION
+LABEL   org.label-schema.build-date=$BUILD_DATE \
+        org.label-schema.name="TreeMaker Docker image for ServiceX" \
+        org.label-schema.description="Provide completely offline-runnable CMSSW images with the TreeMaker and ServiceX dependencies pre-installed." \
+        org.label-schema.url="https://github.com/TreeMaker/TreeMaker" \
+        org.label-schema.vcs-ref=$VCS_REF \
+        org.label-schema.vcs-url=$VCS_URL \
+        org.label-schema.vendor="FNAL" \
+        org.label-schema.version=$VERSION \
+        org.label-schema.schema-version="1.0"
+
+USER    cmsusr
+WORKDIR /home/cmsusr
+
+ARG CMSSW_VERSION=CMSSW_10_2_21
```

Comment:
in the future, we'll have to remember to do a global find/replace whenever changing the CMSSW version...

Reply:
True. It's getting more complicated. The one you're pointing to is a build_arg, so it can be overridden by the .gitlab-ci.yml file, which is where the change ultimately needs to be. We also have a somewhat built-in safety mechanism here because we'd need to build a standalone CMSSW version for any new release.